### PR TITLE
CircleCI

### PR DESCRIPTION
**Pull request by joshaber**

### CODE REVIEW

**Comment by shiftkey** [comment]
@jsohaber maybe need to update [this environment variable?](https://github.com/desktop/desktop/blob/master/app/webpack.production.js#L12)

**Comment by zhouhaoyang** [comment]
@joshaber 

Sorry to disturb you : )

Is there documentation that teach us how to build windows release from desktop source code ?

Thank you very much!

**Comment by RhysBower** [comment]
@zhouhaoyang  You can find [setup documentation](https://github.com/desktop/desktop/blob/master/docs/contributing/setup.md) along with other information in the `docs/` directory of the repo.

**Comment by zhouhaoyang** [comment]
@RhysBower :) Thank you so much!!

File: app/test/unit/git/remote-test.ts [hunk 1]
```diff
@@ -20,12 +20,17 @@ describe('git/remote', () => {
       const testRepoPath = setupFixtureRepository('repo-with-multiple-remotes')
       const repository = new Repository(testRepoPath, -1, null, false)
 
-      const url = 'https://github.com/shiftkey/friendly-bassoon.git'
+      // NB: We don't check for exact URL equality because CircleCI's git config
+      // rewrites HTTPS URLs to SSH.
+      const nwo = 'shiftkey/friendly-bassoon.git'
 
       const result = await getRemotes(repository)
```
Comment (ricksentam) [inline app/test/unit/git/remote-test.ts]: void && do{const result = self.getRemotes(repository)

**Comment by joshaber** [comment]
Let's do this 🤘

**Pull request merged**